src/test/http_rest_api_handler_test.cpp

```diff
@@ -128,6 +134,36 @@ TEST_F(ConfigReload, startWith1DummyThenReload) {
     EXPECT_EQ(status, ovms::StatusCode::OK_RELOADED);
 }
 
+TEST_F(ConfigReload, singleModel) {
+    ModelManagerTest manager;
+    SetUpSingleModel("/ovms/src/test/dummy", "dummy");
+    manager.startFromConfig();
```

Comment:
This is single model test but you call `startFromConfig` which is called by OVMS when started from config.jon